.eslintrc.js

```diff
@@ -1,12 +1,11 @@
 module.exports = {
-  extends: '@mate-academy/eslint-config',
+  extends: "@mate-academy/eslint-config",
   env: {
-    jest: true
+    jest: true,
   },
   rules: {
-    'no-proto': 0,
-    strict: 0,
-    'max-len': 0,
+    "no-proto": 0,
+    quotes: ["off"],
```

Comment:
The `quotes` rule is set to 'off', which means no specific quote style is enforced. If you want to enforce a specific quote style (e.g., single or double quotes), consider setting this rule to ['error', 'single'] or ['error', 'double'].